pom.xml

```diff
@@ -80,6 +80,8 @@
 
     <feature.directory>src/main/feature/feature.xml</feature.directory>
     <spotless.version>2.0.3</spotless.version>
+
+    <m2e.jdt.annotationpath>target/dependency</m2e.jdt.annotationpath>
```

Comment:
Perhaps it's better to move this to the POMs in bundles and itests because there won't be any unpacked EEAs in other projects such as bom/features.

Reply:
done